stable/bookstack/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.podSecurityPolicy.enabled }}
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "bookstack.fullname" . }}
+  labels:
+    app: {{ template "bookstack.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  privileged: false
+  allowPrivilegeEscalation: true
```

Comment:
are you sure you this `allowPrivilegeEscalation` enabled ?

Reply:
Must have been a leftover from my testing, good catch! Fixed!